src/app/charts/catalogservice/templates/pdb.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: policy/v1
+kind: PodDisruptionBudget
+metadata:
+  name: {{ .Chart.Name }}-pdb
+spec:
+  minAvailable: 50%
```

Comment:
why here 50% and for the others absolute numbers?

Reply:
For the healthservice we've only 2 replicas to compensate a single node outage as the expected load is low. Therefore would a single replica in voluntary disruption scenarios be sufficient. 

For catalogservice a failback from let's say 10 (scaled out automatically) nodes to 1 would cause issues, 50% would still result in a significant performance degradation, but the service would remain to work, and 50% is just a starting point to balance between overhead and availability.

For backgroundprocessor we, from my pov, don't need a pdb at all given it has no user impact. But a pdb with minavailable 1 would result in at least 1 remaining replica to process data.

Reply:
ok makes sense